.github/workflows/labels.yml

```diff
@@ -1,4 +1,22 @@
 name: Labels
+
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions
+permissions:
+  actions: none
+  checks: none
+  contents: none
+  deployments: none
+  id-token: none
+  # This action can update/close issues
+  issues: write
+  discussions: none
+  packages: none
+  pages: none
+  pull-requests: none
```

Comment:
Do we not want to enable `write` permissions for pull requests here or is adding a label here not considered a write action?

Reply:
Since labels applies to issue and pull requests I believe this does need `write` access as some labels can `close` the PR as well as comment on it

Reply:
Confirmed :+1: